messages/off-chain-registry/message_manager.go

```diff
@@ -0,0 +1,105 @@
+package offchainregistry
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	"github.com/ava-labs/awm-relayer/config"
+	"github.com/ava-labs/awm-relayer/vms"
+	teleporterregistry "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/upgrades/TeleporterRegistry"
+	"github.com/ethereum/go-ethereum/common"
+	"go.uber.org/zap"
+)
+
+var (
+	OffChainRegistrySourceAddress = common.HexToAddress("0x0000000000000000000000000000000000000000")
```

Comment:
nit: could be `common.Address{}`

Reply:
Yes, but I think it's a bit clearer that this is the zero address and not an uninitalized value. If we could make this a const, then I'd be on board but unfortunately common.Address is not a const-compatible type.